package.json

```diff
@@ -15,8 +15,8 @@
     "test:unit": "vitest --environment jsdom --root .",
     "test:unit:ci": "vitest run --environment jsdom --coverage",
     "test:e2e": "INSTRUMENT_COVERAGE=true start-server-and-test preview :4173 'cypress open --e2e'",
-    "test:e2e:ci": "INSTRUMENT_COVERAGE=true start-server-and-test 'VITE_USE_PROXYURL=false vite dev --port 4173' :4173 'cypress run --browser chrome --e2e'",
-    "test:e2e:dev": "INSTRUMENT_COVERAGE=true start-server-and-test 'vite dev --port 4173' :4173 'cypress open --browser chrome --e2e'",
+    "test:e2e:ci": "INSTRUMENT_COVERAGE=true start-server-and-test 'VITE_USE_PROXYURL=false vite dev --port 4173' :4173 'cypress run --browser chromium --e2e'",
```

Comment:
why this change???

Reply:
Ok seen together: I didn't know you couldn't run the e2e tests because your are using chromium. But apparently you have found a solution. Thx

Reply:
yeah, it was not intended to be merged without notice. I wanted the change to go into CI first, with its share of surprises.
Now, cypress is configured to use the first browser among chrome and chromium it will encounter. That should be pretty robust